MovieQuiz/Presentation/MovieQuizPresenter.swift

```diff
@@ -0,0 +1,127 @@
+import UIKit
+
+final class MovieQuizPresenter: QuestionFactoryDelegate {
+    private let questionsAmount: Int = 10
+    private var currentQuestionIndex: Int = 0
+    
+    private var currentQuestion: QuizQuestion?
+    private var viewController: MovieQuizViewControllerProtocol?
+    
+    private var correctAnswers: Int = 0 //
+    private var questionFactory: QuestionFactoryProtocol? = nil//
```

Comment:
для переменной инициализация избыточна, компилятор сам установит nil
```suggestion
    private var questionFactory: QuestionFactoryProtocol? 
```

Reply:
ok